chunk/chunk_store.go

```diff
@@ -165,29 +116,20 @@ type PeriodicTableConfig struct {
 type AWSStore struct {
 	cfg StoreConfig
 
-	dynamoRequests     chan dynamoOp
-	dynamoRequestsDone sync.WaitGroup
+	dynamo *dynamoDBBackoffClient
 }
 
 // NewAWSStore makes a new ChunkStore
-func NewAWSStore(cfg StoreConfig) (*AWSStore, error) {
-	store := &AWSStore{
-		cfg:            cfg,
-		dynamoRequests: make(chan dynamoOp),
-	}
-
-	store.dynamoRequestsDone.Add(numDynamoRequests)
-	for i := 0; i < numDynamoRequests; i++ {
-		go store.dynamoRequestLoop()
+func NewAWSStore(cfg StoreConfig) *AWSStore {
+	return &AWSStore{
+		cfg:    cfg,
+		dynamo: newDynamoDBBackoffClient(cfg.DynamoDB),
```

Comment:
Maybe don't even do the backoff bit in here, but rather leave that up to the caller to construct.